### ISSUE TITLE
Update docs to include references to all new Tensorflow loss functions and new Models

### ISSUE THREAD

**Issue by jlema** [microsoft/BackwardCompatibilityML#93]
https://backwardcompatibilityml.readthedocs.io/en/latest/backwardcompatibilityml/topics/getting_started.html#backward-compatibility-ml-library-examples-included

https://backwardcompatibilityml.readthedocs.io/en/latest/backwardcompatibilityml/topics/integrating_loss_functions.html

Plus a new topic for the Model workflow
